docs/source/getting_started/installation/index.md

```diff
@@ -0,0 +1,37 @@
+## Installing PyMC-Marketing
```

Comment:
Can you note installing from GitHub directly via pip for use of latest functionality

Reply:
done!